programs/ssl/ssl_server2.c

```diff
@@ -1041,8 +1056,8 @@ int main( int argc, char *argv[] )
         }
         else if( strcmp( p, "renego_period" ) == 0 )
         {
-            opt.renego_period = atoi( q );
-            if( opt.renego_period < 2 || opt.renego_period > 255 )
+            if( sscanf( q, "%" SCNu64, &opt.renego_period ) != 1 ||
```

Comment:
Indeed, the portable formatting macros were introduced in C99. I will try and find out another way of parsing a 64-bit integer, but I am not sure whether I will find a portable way to do it. Also bear in mind that this is the example code.